specs/protocol/jovian/exec-engine.md

```diff
@@ -0,0 +1,72 @@
+# Jovian: Execution Engine
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Minimum Base Fee](#minimum-base-fee)
+  - [Minimum Base Fee in Block Header](#minimum-base-fee-in-block-header)
+  - [Minimum Base Fee in `PayloadAttributesV3`](#minimum-base-fee-in-payloadattributesv3)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Minimum Base Fee
+
+Jovian introduces a
+[configurable minimum base fee](https://github.com/ethereum-optimism/design-docs/blob/main/protocol/minimum-base-fee.md)
+to reduce the duration of priority-fee auctions on OP Stack chains.
+
+The minimum base fee is configured via `SystemConfig` (see `./system-config.md`) and enforced by the execution engine
+via the block header `extraData` encoding and the Engine API `PayloadAttributesV3` parameters.
+
+### Minimum Base Fee in Block Header
+
+Like [Holocene's dynamic EIP-1559 parameters](../holocene/exec-engine.md#dynamic-eip-1559-parameters), Jovian encodes
+fee parameters in the `extraData` field of each L2 block header. The format is extended to include an additional byte
+for the minimum base fee exponent.
+
+| Name                | Type               | Byte Offset |
+| ------------------- | ------------------ | ----------- |
+| `minBaseFeeFactors` | `uint8`            | `[9, 10)`   |
+
+Constraints:
+
+- `version` MUST be `1` (incremented from Holocene's `0`).
+- There MUST NOT be any data beyond these 10 bytes.
+
+The `minBaseFeeFactors` field encodes minimum base fee in wei as a 4-bit significand followed by a
+4-bit exponent to the power of 10. A value of `0` disables the minimum base fee entirely.
+
+```javascript
+if minBaseFeeFactors > 0 {
```

Comment:
I think with this new encoding, it would be possible to not have this outer `if` statement. Any `minBaseFeeFactors` value which implies a `0` significand would be sufficient to effectively disable the feature. That said, I'm fine with keeping it if you think it is clearer.